states/statefile/testdata/roundtrip/v3-bigint.in.tfstate

```diff
@@ -0,0 +1,136 @@
+{
+    "version": 3,
+    "terraform_version": "0.11.1",
+    "serial": 8,
+    "lineage": "0f5b2ff9-6ff5-8e9e-1f81-aa3ce9a483eb",
+    "backend": {
+        "hash": 10669755453527594976
+    },
```

Comment:
This file was copied from v3-grabbag.in.tfstate and the following lines are added.

```
[terraform@fix-statev2-hash-decode|✔]$ diff -u states/statefile/testdata/roundtrip/{v3-grabbag.in.tfstate,v3-bigint.in.tfstate}
--- states/statefile/testdata/roundtrip/v3-grabbag.in.tfstate   2018-12-13 11:18:13.000000000 +0900
+++ states/statefile/testdata/roundtrip/v3-bigint.in.tfstate    2019-05-28 23:56:58.000000000 +0900
@@ -3,6 +3,9 @@
     "terraform_version": "0.11.1",
     "serial": 8,
     "lineage": "0f5b2ff9-6ff5-8e9e-1f81-aa3ce9a483eb",
+    "backend": {
+        "hash": 10669755453527594976
+    },
     "modules": [
         {
             "path": [
```